BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmySecondViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  JimmySecondViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+protocol canRecieve {
+    func dataRecieved(data: String)
+}
+
+class JimmySecondViewController: UIViewController {
+    
+    var delegate : canRecieve?
```

Comment:
Let me introduce you to memory leaks. Let me know if you see why you have memory leak here and how you'd fix it.

Also the space should only go after the `:`. Like this: 
```suggestion
    var delegate: canRecieve?
```